packages/mds-compliance-engine/tests/engine/engine-test.spec.ts

```diff
@@ -38,9 +39,9 @@ let policies: PolicyDomainModel[] = []
 
 const CITY_OF_LA = '1f943d59-ccc9-4d91-b6e2-0c5e771cbc49'
 
-const geographies: Geography[] = [
+const geographies = [
   { name: 'la', geography_id: CITY_OF_LA, geography_json: LA_CITY_BOUNDARY as FeatureCollection }
-]
+] as GeographyDomainModel[]
```

Comment:
Why the change to `as` casting?

Reply:
for brevity, to quickly convert these fixtures to `GeographyDomainModel`, without appending the required-but-nullable fields.